--- flac-1.2.1/configure.in	2007-09-13 17:48:42.000000000 +0200
+++ flac-1.2.1/configure.in	2009-11-17 12:03:34.000000000 +0100
@@ -85,7 +85,7 @@
 
 # only needed because of ntohl() usage, can get rid of after that's gone:
 case "$host" in
-	*-*-cygwin|*mingw*) MINGW_WINSOCK_LIBS=-lwsock32 ;;
+	*-*-cygwin|*mingw*) MINGW_WINSOCK_LIBS= ;;
 	*) MINGW_WINSOCK_LIBS= ;;
 esac
 AC_SUBST(MINGW_WINSOCK_LIBS)
