--- include/cppunit/TestAssert.h.orig	2012-08-01 14:17:55.299104325 +1000
+++ include/cppunit/TestAssert.h	2012-08-01 14:20:25.009232691 +1000
@@ -76,7 +76,7 @@
        const int precision = 15;
 #endif  // #ifdef DBL_DIG
        char buffer[128];
-#ifdef __STDC_SECURE_LIB__ // Use secure version with visual studio 2005 to avoid warning.
+#if defined(MSC_VER) && defined(__STDC_SECURE_LIB__) // Use secure version with visual studio 2005 to avoid warning.
        sprintf_s(buffer, sizeof(buffer), "%.*g", precision, x); 
 #else	
        sprintf(buffer, "%.*g", precision, x); 
